.github/workflows/ci.yml

```diff
@@ -131,8 +131,8 @@ jobs:
         uses: friendlyanon/setup-vcpkg@v1
         with:
           path: src/realm-core/tools/vcpkg/ports
-          cache-key: ${{ matrix.preset }}
-          cache-restore-keys: ${{ matrix.preset }}
+          cache-key: vcpkg-${{ matrix.preset }}-${{ hashFiles('./src/realm-core/tools/vcpkg/**') }}
```

Comment:
Rather than hashing the contents of vcpkg, can we use the submodule sha instead?

Reply:
There isn't a clear-cut way I can get that outside of a multi-line git invocation, I think. I can just fall back to `github.sha` - how does that sound?